interfaces/cython/cantera/onedim.pyx

```diff
@@ -385,12 +413,18 @@ cdef class ReactingSurface1D(Boundary1D):
             raise TypeError('Kinetics object must be derived from '
                             'InterfaceKinetics.')
         self.surf.setKineticsMgr(<CxxInterfaceKinetics*>kin.kinetics)
+        self._surface = kin
+        self._surface._references[self._weakref_proxy2] = True
 
     property coverage_enabled:
         """Controls whether or not to solve the surface coverage equations."""
         def __set__(self, value):
             self.surf.enableCoverageEquations(<cbool>value)
 
+    property surface:
+        def __get__(self):
+            return self._surface
+
```

Comment:
If this is just going to be a simple passthrough, you can just declare it as `cdef public Kinetics surface` in `_cantera.pxd`.